presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -479,9 +479,12 @@ else if (!partitionColumnNames.contains(name)) {
         table.setPartitionKeys(partitionColumns);
         table.setSd(sd);
 
-        PrivilegeGrantInfo allPrivileges = new PrivilegeGrantInfo("all", 0, tableOwner, PrincipalType.USER, true);
+        PrivilegeGrantInfo selectPrivilege = new PrivilegeGrantInfo("select", 0, tableOwner, PrincipalType.USER, true);
```

Comment:
I see. Was this a recent change? I couldn't find any mention of this here: https://cwiki.apache.org/confluence/display/Hive/SQL+Standard+Based+Hive+Authorization